courses/machine_learning/deepdive2/time_series_prediction/4_modeling_keras.ipynb

```diff
@@ -0,0 +1,1261 @@
+{
```

Comment:
<p>import datetime</p><p>import shutil</p><p> </p><p>these are unnecessary, see comments below.</p><p> </p><p>can you please add the following lines?</p><p> </p><p># For reproducible results.</p><p>from numpy.random import seed</p><p>seed(1)</p><p>from tensorflow import set_random_seed</p><p>set_random_seed(2)</p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/689/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Done. </p><p>Note, <code>tf.set_random_seed(seed)</code>&nbsp;has changed to&nbsp;<code>tf.random.set_seed(seed)</code>&nbsp;in TensorFlow 2.</p>